lib/shopify_transporter/pipeline/magento/product/top_level_attributes.rb

```diff
@@ -45,11 +45,11 @@ def attributes_from(input)
               attributes = map_from_key_to_val(COLUMN_MAPPING, input)
               attributes['published'] = published?(input)
               attributes['published_scope'] = published?(input) ? 'global' : ''
-              attributes['published_at'] = published?(input) ? input['updated_at'] : ''
+              attributes['published_at'] = published?(input) ? input['updated_at'] : nil
```

Comment:
Why not just:
```ruby
attributes['published_at'] = input['updated_at'] if published?(input)

Reply:
huh idk how i didnt think of that. thanks